modules/markdown/markdown.go

```diff
@@ -94,6 +95,64 @@ var (
 	// FIXME: this pattern matches pure numbers as well, right now we do a hack to check in RenderSha1CurrentPattern
 	// by converting string to a number.
 	Sha1CurrentPattern = regexp.MustCompile(`\b[0-9a-f]{40}\b`)
+
+	WikiLinkPattern = regexp.MustCompile(`(\[\[.*\]\]\w*)`)
+
+	/*
```

Comment:
@metalmatze I can't quite catch a meaning. It is necessary for the system to work.